tracer/src/Datadog.Trace/Logging/DirectSubmission/Formatting/LogFormatter.cs

```diff
@@ -333,10 +333,20 @@ internal void FormatCIVisibilityLog(StringBuilder sb, string source, string? log
                 }
 
                 writer.WritePropertyName("dd.trace_id", escape: false);
-                writer.WriteValue($"{span.TraceId}");
+
+                if (span is Span { TraceId128.Upper: > 0 } s)
```

Comment:
Should this also be behind the 128-bit logging feature flag?

Reply:
Definitely!

Reply:
done